src/component/Bpanel.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+
+const Bpanel = () => (
+
+  <div className="container">
+    <div className="row ">
+      <div className="col-3">
+        <button type="button" className="btn btn-success">AC</button>
```

Comment:
This should be rendered with a `Button` component not a html button

Reply:
That means we need to build a button component

![Screenshot 2019-12-13 at 4 03 21 PM](https://user-images.githubusercontent.com/10922508/70809531-3b103400-1dc2-11ea-8e53-b6737b5e0cec.png)